engine/crates/integration-tests/tests/tracing/v1.rs

```diff
@@ -106,20 +124,29 @@ async fn query_named() {
         .with_filter(|meta| meta.is_span() && meta.target() == "grafbase" && *meta.level() >= Level::INFO)
         .new_span(graphql_span.clone())
         .enter(graphql_span.clone())
+        .new_span(
+            resolver_span
+                .clone()
+                .with_field(expect::field("resolver.name").with_value(&"test")),
+        )
+        // Who knows why we're doing that
```

Comment:
git blame?

Reply:
I meant I don't know why we're entering & exiting the resolver span, but tracing mock is tricky. It tracks everything that is happening to a span, meaning also when tokio decides to switch to a different future.... so who knows 🤷